### PR TITLE
Improve texts

### PR DESCRIPTION
**Pull request by cadamini**
### Description

While reading the article I found some places that I think should be improved/shortened/reorganized.

### Motivation

I love writing. Sometimes, things are more complex than needed.

### CODE REVIEW

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 1]
```diff
@@ -123,7 +123,9 @@ You should now see that you have flexible tracks. The `fr` unit distributes spac
 }
 ```
 
-The first track now gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger. You can mix `fr` units with fixed length units — in such a case the space needed for the fixed tracks is used up first; the remaining space is then distributed to the other tracks.
+The first track gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:126]:
```suggestion
The first track gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger.  
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 2]
```diff
@@ -123,7 +123,9 @@ You should now see that you have flexible tracks. The `fr` unit distributes spac
 }
 ```
 
-The first track now gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger. You can mix `fr` units with fixed length units — in such a case the space needed for the fixed tracks is used up first; the remaining space is then distributed to the other tracks.
+The first track gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger. 
+
+You can mix `fr` units with fixed length units — in such a case the space needed for the fixed tracks is used up first; the remaining space is then distributed to the other tracks.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:128]:
```suggestion
You can mix `fr` units with fixed length units — in such a case the space needed for the fixed tracks is used up first; the remaining space is then distributed to the other tracks.  
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 3]
```diff
@@ -159,7 +161,10 @@ body {
 
 ### Gaps between tracks
 
-To create gaps between tracks we use the properties {{cssxref("column-gap")}} for gaps between columns, {{cssxref("row-gap")}} for gaps between rows, and {{cssxref("gap")}} as a shorthand for both.
+To create gaps between tracks, we use the properties:
+- {{cssxref("column-gap")}} for gaps between columns
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:165]:
```suggestion

- {{cssxref("column-gap")}} for gaps between columns
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 4]
```diff
@@ -229,7 +234,11 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:237]:
```suggestion
We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.  
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 5]
```diff
@@ -229,7 +234,11 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. 
+
+The difference:
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:239]:
```suggestion
The difference:  
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 6]
```diff
@@ -229,7 +234,11 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. 
+
+The difference: 
+- Explicit grid: Created using `grid-template-columns` or `grid-template-rows`.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:240]:
```suggestion

- Explicit grid: Created using `grid-template-columns` or `grid-template-rows`.  
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 7]
```diff
@@ -123,7 +123,9 @@ You should now see that you have flexible tracks. The `fr` unit distributes spac
 }
 ```
 
-The first track now gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger. You can mix `fr` units with fixed length units — in such a case the space needed for the fixed tracks is used up first; the remaining space is then distributed to the other tracks.
+The first track gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:126]:
```suggestion
The first track gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger.
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 8]
```diff
@@ -229,7 +235,12 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:238]:
```suggestion
We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 9]
```diff
@@ -229,7 +235,12 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.  
+
+The difference:
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:240]:
```suggestion
The difference:
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 10]
```diff
@@ -229,7 +235,12 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.  
+
+The difference:  
+
+- Explicit grid: Created using `grid-template-columns` or `grid-template-rows`.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:242]:
```suggestion
- Explicit grid: Created using `grid-template-columns` or `grid-template-rows`.
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 11]
```diff
@@ -229,7 +235,12 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.  
+
+The difference:  
+
+- Explicit grid: Created using `grid-template-columns` or `grid-template-rows`.  
+- Implicit grid: Extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:243]:
```suggestion
- Implicit grid: Extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 12]
```diff
@@ -114,7 +114,7 @@ Change your track listing to the following definition, creating three `1fr` trac
 }
 ```
 
-You should now see that you have flexible tracks. The `fr` unit distributes space proportionally. So if you specify different positive values for your tracks like so:
+You should have flexible tracks. The `fr` unit distributes space proportionally. You can specify different positive values for your tracks like so:
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:117]:
```suggestion
You now have flexible tracks. The `fr` unit distributes space proportionally. You can specify different positive values for your tracks like so:
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 13]
```diff
@@ -123,7 +123,9 @@ You should now see that you have flexible tracks. The `fr` unit distributes spac
 }
 ```
 
-The first track now gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger. You can mix `fr` units with fixed length units — in such a case the space needed for the fixed tracks is used up first; the remaining space is then distributed to the other tracks.
+The first track gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger.
+
+You can mix `fr` units with fixed length units — in such a case the space needed for the fixed tracks is used up first; the remaining space is then distributed to the other tracks.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:128]:
```suggestion
The first track gets `2fr` of the available space and the other two tracks get `1fr`, making the first track larger. You can mix `fr` units with fixed length units. In this case, the space needed for the fixed tracks is used up first before the remaining space is distributed to the other tracks.
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 14]
```diff
@@ -159,7 +161,11 @@ body {
 
 ### Gaps between tracks
 
-To create gaps between tracks we use the properties {{cssxref("column-gap")}} for gaps between columns, {{cssxref("row-gap")}} for gaps between rows, and {{cssxref("gap")}} as a shorthand for both.
+To create gaps between tracks, we use the properties:
+
+- {{cssxref("column-gap")}} for gaps between columns
+- {{cssxref("row-gap")}} for gaps between rows
+- {{cssxref("gap")}} as a shorthand for both.
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:168]:
```suggestion
- {{cssxref("gap")}} as a shorthand for both
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 15]
```diff
@@ -201,7 +207,7 @@ body {
 
 {{ EmbedLiveSample('Gaps_between_tracks', '100%', 400) }}
 
-> **Note:** The `gap` properties (`column-gap`, `row-gap` and `gap`) used to be prefixed by `grid-`, but this has been changed in the spec in order to make them usable in multiple layout methods. The prefixed versions will be maintained as an alias, so they'll be safe to use for some time. To be on the safe side, you could double up and add both properties to make your code more bulletproof:
+> **Note:** The `gap` properties (`column-gap`, `row-gap` and `gap`) used to be prefixed by `grid-`. This has been changed in the spec but the prefixed versions will be maintained as an alias. To be on the safe side, you could double up and add both properties to make your code more bulletproof:
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:210]:
```suggestion
> **Note:** The `gap` properties (`column-gap`, `row-gap` and `gap`) used to be prefixed by `grid-`. The spec has changed but the prefixed versions will be maintained as an alias. To be on the safe side and make your code more bulletproof, you can add both properties:
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 16]
```diff
@@ -229,7 +235,12 @@ You'll now get three `1fr` tracks just as before. The first value passed to the
 
 ### The implicit and explicit grid
 
-We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid. The explicit grid is the one that you create using `grid-template-columns` or `grid-template-rows`. The implicit grid extends the defined explicit grid when content is placed outside of that grid, such as into our rows by drawing additional grid lines.
+We've only specified column tracks so far, yet rows are being created to hold our content. This is an example of the _explicit_ versus the _implicit_ grid.
+
+The difference:
```
Comment (cadamini) [inline files/en-us/learn/css/css_layout/grids/index.md:240]:
```suggestion
Difference between explicit and implicit grid:
```

**Review by dipikabh — APPROVED** [review]
The changes look good. Thanks @cadamini 👍

**Pull request merged**